### DISCUSSION TITLE
How to display latest posts first in blog page ?

### DISCUSSION THREAD

**Discussion by sid-r-singh** [adityatelange/hugo-PaperMod#424] [Q&A]
Is there any way to display posts in blog page in decreasing order of date?

**Comment by icy-comet** [comment]
Disable the theme's profile mode. The default view lists the posts just as you need.

**Reply by sid-r-singh** [reply]
Oh.. no. I love using the profile mode. So now I have a button to my blog. Is there any way for me to display posts in blog page in decreasing order of date?

**Reply by icy-comet** [reply]
You could always override the default templates and files and customize as you want. Just replicate the directory structure of the theme's `layout` folder in yours and you should be good to go.

**Comment by kszpieg** [comment]
Did someone successfully customize the theme in this way? I mean some kind of merging ProfileMode and the default list of posts under the picture? Maybe @adityatelange can provide some hints :D
